### DISCUSSION TITLE
Link in the email without :8055

### DISCUSSION THREAD

**Discussion by koksikus** [directus/directus#13350] [Archive]
Hi, 
How to remove fragment from domain: 8055. When Directus sends me an email with a notification, it uses the domain http: // url: 8055 / admin / content / Workstations / 2 which fails. All I have to do is remove: 8055 in link in email and it's already a success ...

**Comment by azrikahar** [comment]
Make sure to set the `PUBLIC_URL` environment variable to your desired base URL (eg. `https://example.com`) 👍 

Just to share a more in depth answer, it is being used in line 87 here:

https://github.com/directus/directus/blob/726af0c57cf4e3a94c61d828db0dbd811851c4c9/api/src/services/activity.ts#L80-L90

**Comment by koksikus** [comment]
Thank you very much.
